aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -955,6 +961,126 @@ void qtopk_kernel(Tensor& values,
   });
 }
 
+template <bool ReluFused>
+void q_batch_norm_kernel(
+    int N,
+    int C,
+    int HxW,
+    const int in_zero_point,
+    const int out_zero_point,
```

Comment:
zero points should definitely be `int64_t`, right?

Reply:
Yeah, updated them